geniuspaste/src/geniuspaste.c

```diff
@@ -94,6 +94,54 @@ PLUGIN_SET_TRANSLATABLE_INFO(LOCALEDIR, GETTEXT_PACKAGE, PLUGIN_NAME,
                              PLUGIN_VERSION, "Enrico Trotta <enrico.trt@gmail.com>")
 
 
+/* Windows compatibility for resolving system plugin data path.
+ * This only works if the plugin's data is installed in the same tree as the
+ * plugin DLL, which might not be true (as the DLL has to be in Geany's libdir
+ * but the plugin's data might be anywhere in theory) */
```

Comment:
How can we solve this better?  I have no idea how we could locate the data files if they are not in the same tree as the plugin's DLL.  I don't know if it's a plausible situation on Windows, but it is on Linux, so it's at least worth asking.